src/main.rs

```diff
@@ -417,21 +426,25 @@ fn print_avgs(options: &Options, avgs: &BTreeMap<String, avg::AVG>) {
             _ => {
                 if !options.upgradable_only {
                     if options.quiet > 0 {
-                        println!("{}", pkg);
+                        t.fg(avg.severity.to_color()).expect("term::fg failed");
+                        writeln!(t, "{}", pkg).expect("term::writeln failed");
+                        t.reset().expect("term::stdout failed");
                     } else {
                         match options.format {
-                            Some(ref f) => println!(
-                                "{}",
-                                f.replace("%n", pkg.as_str())
-                                    .replace("%c", avg.issues.iter().join(",").as_str(),)
-                            ),
+                            Some(ref f) => {
+                                t.fg(avg.severity.to_color()).expect("term::fg failed");
```

Comment:
CVEs are still coloured in this case.

Reply:
I didn't change the behavior to keep things simple and still colored, but if you want i can remove the color for format string.